src/components/@shared/DDODownloadButton/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactElement } from 'react'
+import Button from '../atoms/Button'
+
+export default function DDODownloadButton({
+  asset
+}: {
+  asset: any
+}): ReactElement {
+  // Download DDO as JSON
+  const downloadJson = () => {
```

Comment:
suggestion(blocking): it could make sense to check the `downloadFileBrowser` function that is exported by the ocean.js library

If that function is not working for our use-case, we should separate the `downloadJSON` function to be re-usable utility in the repository that accepts any JSON valid object and downloads it as a file.